### ISSUE TITLE
Marcato Articulation in Gonville Style

### ISSUE THREAD

**Issue by ConnorAMcKenna** [musescore/MuseScore#19946]
### Issue type

Engraving bug

### Bug description

Gonville has a unique design for the marcato mark compared to the other musical styles. However, when using combined articulations (staccato-marcato, tenuto-marcato, etc.) from palettes, the NORMAL marcato mark appears (as it is in other styles). This means that if a piece uses marcato and combined marcato articulations, there are two unique designs for marcato within a score (the normal style and the Gonville style) - this is very confusing. I think the normal marcato should be available in Gonville to use by itself (as opposed to only in a combined articulation). Note: this applies only to adding combined articulations from palettes, not by using key commands to add them individually.

### Steps to reproduce

1. Add several notes in a score
2. Attach marcato to to one
3. Attach a combined articulation from palettes

### MuseScore Version

OS: macOS 14.1, Arch.: x86_64, MuseScore version (64-bit): 4.1.1-232071203, revision: github-musescore-musescore-e4d1ddf

### Regression

I don't know

### Operating system

macOS Sonoma 14.1

**Comment by oktophonie** [comment]
The reason this happens: SMuFL defines articulations as separate glyphs and also as combined glyphs where appropriate, including the marcato-staccato, for example (see: https://w3c.github.io/smufl/latest/tables/articulation.html).

However, the Gonville font only has the separate articulations, not the combined ones. The standard behaviour when a glyph is unavailable in the chosen music font is to fall back to using Bravura instead, which is what is happening in your case, hence the different symbols being used. (If you look closely you'll see the staccato is different too, though that's a lot less obvious!)

The combined articulations could certainly be added to Gonville; however, the behaviour of combined articulations was rationalised in 4.1 and 4.2 so that these combined symbols are almost never used in practice, which allows for much more intelligent placement of the symbols. For example, a staccato or tenuto inside another symbol can now go into a stave space, which would never be possible with a combined symbol.

We didn't want to make it completely impossible to enter the combined symbols though, since users may have a specific reason for wanting them, so it's still possible to obtain them via the palette. But that does give rise to this possible confusion, so this issue is a good reminder that we'd meant to shuffle those combined articulations into the 'More' section of the palette so people would be less tempted to use them, but we forgot.

In summary:

- when you want multiple articulations on a note, always enter them separately (which is the only way you can do it anyway from the top toolbar); this gives MuseScore more flexibility about the placement of the marks and also sidesteps the problem of different fonts being used.
- we should move the combined artics into 'More'
- we should, at some point, add combined artic symbols to Gonville just in case

**Comment by Jojo-Schmitz** [comment]
Adding those glyphs to Gonville might be a task for @Gootector, thr creator of that font

**Comment by Gootector** [comment]
This font is/was designed for MuseScore 2, when MuseScore was real-open-source software. Now, Muse Group hires employees. Unfortunately, I am not one of them. I can translate the program, I can fix small text issues in the program. That's all I can do for MuseScore (for free).

Greetings,
Gootector

**Comment by Jojo-Schmitz** [comment]
It still is Open Source, and contributions from non-staff members are still taken. Even as Mu2 times there were (paid-for) staff members (Werner, Nicolas and Thomas were not living from thin air, Dmitry and Anatoly neither)

**Comment by Gootector** [comment]
> It still is Open Source, and contributions from non-staff members are still taken. Even as Mu2 times there were (paid-for) staff members (Werner, Nicolas and Thomas were not living from thin air, Dmitry and Anatoly neither).
> 
> But of course whether or nor **you** contribute is at your discretion.

Why should I contribute to the MuseScore project for free... if someone gets paid for the same work?
Of course, MuseScore is STILL open-source, because MU complies with the guidelines of open-source. Otherwise I wouldn't be here 😉
